### DISCUSSION TITLE
What are the rules for transferring an ordinal/inscription? What breaks it and what doesn't?

### DISCUSSION THREAD

**Discussion by andrewhong5297** [ordinals/ord#1687] [Q&A]
Would be good to know when an inscription is considered "broken" or not. If I spend an inscribed ordinal to create a non-P2TR output (P2WPKH maybe), but then use that output to create a P2TR output in the next transaction, does the inscription survive? If it went through a coinjoin pool, does the inscription survive? If I inscribe on an ordinal output that already contains an inscription, is the original overridden or are they combined?

It would be great to have more guides here on ordinal + inscription tracking. I'm happy to write one; I already write [a lot of data guides](https://web3datadegens.substack.com/) for the crypto space.

**Comment by raphjaph** [comment]
Ordinal theory is a way to track single satoshis across UTXOs. Once a satoshi has been inscribed it is stamped forever. There is no way to get rid of that attachment through CoinJoins or using scripts other thatn P2TR. If an ordinal is reinscribed the convention is to honor the first inscription but we are still exploring that. The tricky part about ordinal theory is arranging the inputs and outputs of a transaction so that the satoshi doesn't accidentally end up as fees. 

PS. there is only one way to destroy a satoshi/ordinals, which is if the miner underpays the coinbase output

**Reply by andrewhong5297** [reply]
Inscription are assigned to the full range of the output right? Can I add another inscription on top if I just inscribe the output again?

Good to know there's no way of destroying them

**Reply by raphjaph** [reply]
Inscriptions are assigned to a **single** sat. Kind of like stamping a penny. 

Through ordinal theory single sats become stable identifiers that have specific properties. One of the properties is having an inscription but there are more. Look at [Sat 1737591307096462](https://ordinals.com/sat/1737591307096462) for example. 

At the moment we do not allow reinscribing sats but it is possible in theory. The second inscription will not show up on the block explorer though.
